src/config/logger.ts

```diff
@@ -17,34 +17,47 @@ const outputMode = process.env.MICROS_ENV ? "json" : "short";
 //
 export const FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME = "frontend-log-middleware";
 // add levelInString to include DEBUG | ERROR | INFO | WARN
-const LOG_STREAM = filteringHttpLogsStream(FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME,
+const LOG_STREAM = filteringHttpLogsStream(
+	FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME,
 	bformat({ outputMode, levelInString: true })
 );
 
-const responseSerializer = (res: AxiosResponse) => ({
+const responseSerializer = (res) => res && ({
```

Comment:
Why remove the typing here and below?

Reply:
Because not all responses are `AxiosResponse` and didn't want to add that assumption in case someone adds something from AxiosResponse in the code because the type said it was fine.